entrypoint

```diff
@@ -51,6 +51,26 @@ if [ ! -e "\$PREF_FILE" ]; then
     source gen-config.sh
 fi
 
+# Changes the log verbosity
+if [ "$LOGVERBOSE" = "1" ] || [ -z "$LOGVERBOSE" ]; then
+    setPref "LogVerbose" "1"
+elif [ "$LOGVERBOSE" = "0" ]; then
+    setPref "LogVerbose" "0"
```

Comment:
As for setting the inverse value, perhaps we should only do that if `LOG_VERBOSE` is set/non-empty.

Following the above, something like this would work I think:
```
if [ -n "$LOG_VERBOSE" ]; then
    if bool "$LOG_VERBOSE"; then
        setPref "LogVerbose" "1"
￼    else
￼         setPref "LogVerbose" "0"
    fi
fi
```
or of course mapping the true/false return value of the bool function to a 0/1 and setting that as the value would probably be better :)

Reply:
Again, I think it's best to stick to Plex built-in values `0` or `1`, and defaults to `1` (as it is now).